tests/integration/conftest.py

```diff
@@ -32,34 +32,48 @@
                                  "RE_JOB_ACTION": "RE_JOB_ACTION",
                                  "RE_JOB_IMAGE": "RE_JOB_IMAGE",
                                  "RE_JOB_SCENARIO": "RE_JOB_SCENARIO",
-                                 "RE_JOB_BRANCH": "RE_JOB_BRANCH",
+                                 "RE_JOB_BRANCH": "pike-rc",
                                  "RPC_RELEASE": "RPC_RELEASE",
                                  "RPC_PRODUCT_RELEASE": "RPC_PRODUCT_RELEASE",
                                  "OS_ARTIFACT_SHA": "OS_ARTIFACT_SHA",
                                  "PYTHON_ARTIFACT_SHA": "PYTHON_ARTIFACT_SHA",
                                  "APT_ARTIFACT_SHA": "APT_ARTIFACT_SHA",
-                                 "REPO_URL": "REPO_URL",
+                                 "REPO_URL": "https://github.com/rcbops/rpc-openstack",
                                  "JOB_NAME": "JOB_NAME",
```

Comment:
These changes were made because of the addition of GitHub linking. Without these changes all the tests would break.